lib/event.ts

```diff
@@ -0,0 +1,28 @@
+import { LogDescription } from '@ethersproject/abi';
+import { ContractReceipt } from 'ethers';
+import { Interface } from 'ethers/lib/utils';
+
+export function getLogs(
+  contractInterface: Interface,
+  receipt: ContractReceipt,
+  only: string[] = [],
```

Comment:
Might as well add `except` here to continue the only/except pattern.  But I'm thinking about making a separate package to handle that logic because it's used it so many places, so we can just wait for that.

Reply:
Absolutely, the filter I did for Diamond utils also supports `except` so it may make the most sense to wait until that's merged or branch from that PR and break out the filters specifically.